core/sqf/src/seatrans/.gitignore

```diff
@@ -12,3 +12,6 @@ hbase-trx/src/main/java/org/apache/hadoop/hbase/regionserver/transactional/Clean
 hbase-trx/src/main/java/org/apache/hadoop/hbase/regionserver/transactional/KeyValueListScanner.java
 hbase-trx/src/main/java/org/apache/hadoop/hbase/regionserver/transactional/MemoryUsageChore.java
 hbase-trx/src/main/java/org/apache/hadoop/hbase/regionserver/transactional/TrxTransactionState.java
+hbase-trx/src/main/java/org/apache/hadoop/hbase/client/ClientScanner98.java
```

Comment:
Is the name ClientScanner98 significant? For example, does it work only on HBase 0.98? Is there a different implementation needed for HBase 1.0?